src/pom.xml

```diff
@@ -1617,6 +1622,7 @@
   <gs.version>2.9-SNAPSHOT</gs.version>
   <gt.version>15-SNAPSHOT</gt.version>
   <gwc.version>1.9-SNAPSHOT</gwc.version>
+  <jodatime.version>2.2</jodatime.version>
```

Comment:
This is just a single jar right? could leave it in the dependency management section ...